TripKit/Classes/helpers/TKUserProfileHelper.swift

```diff
@@ -77,18 +77,20 @@ public class TKUserProfileHelper: NSObject {
   @objc public class func setModeIdentifier(_ modeIdentifier: Identifier, toHidden hidden: Bool) {
     update(hiddenModeIdentifiers, forKey: .hidden, modeIdentifier: modeIdentifier, include: hidden)
     
-    // We give special consideration when modifying walking or wheelchair mode since we may
-    // need to perform toggle, e.g., adding walking while wheelchair is already included
-    // calls for toggle.
-    let isModifyingWalking = modeIdentifier == SVKTransportModeIdentifierWalking
-    let walkingIsCurrentlyIncluded = !modeIdentifierIsHidden(SVKTransportModeIdentifierWalking)
-    let isModifyingWheelchair = SVKTransportModes.modeIdentifierIsWheelchair(modeIdentifier)
-    let wheelchairIsCurrentlyIncluded = !modeIdentifierIsHidden(SVKTransportModeIdentifierWheelchair)
-    
-    if isModifyingWalking && wheelchairIsCurrentlyIncluded {
-      update(hiddenModeIdentifiers, forKey: .hidden, modeIdentifier: SVKTransportModeIdentifierWheelchair, include: !hidden)
-    } else if isModifyingWheelchair && walkingIsCurrentlyIncluded {
-      update(hiddenModeIdentifiers, forKey: .hidden, modeIdentifier: SVKTransportModeIdentifierWalking, include: !hidden)
+    switch (modeIdentifier, hidden) {
```

Comment:
Actually, this is flawed, particularly when we move to another city where wheelchair isn't supported. Since this logic depends on whether wheelchair is supported in a region (and `TKUserProfileHelper` isn't the right class to keep track of that, but whether to show or not to show), decided to move this out of TripKit and to `TransportModeDetailViewModel` in TripGoAppKit.